central/image/service/http_handler.go

```diff
@@ -0,0 +1,153 @@
+package service
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	clusterUtil "github.com/stackrox/rox/central/cluster/util"
+	"github.com/stackrox/rox/central/role/sachelper"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/httputil"
+	"github.com/stackrox/rox/pkg/images/enricher"
+	"github.com/stackrox/rox/pkg/images/integration"
+	"google.golang.org/grpc/codes"
+)
+
+const (
+	NumberBytes = 100 * 1024
+	timeout     = 10 * time.Minute
```

Comment:
Consider replacing this with an env var so that if there are issues it can be changed. Can use the existing `env.ScanTimeout` or creating a new one specifically for SBOMs that has the same default value as `ScanTimeout`.

```suggestion
	sbomGenTimeout     = env.ScanTimeout.Duration()
```

Reply:
done